src/lib/utils/sizes.js

```diff
@@ -78,6 +78,7 @@ const height = getScreenHeight()
 export const isSmallDevice = width < 350
 export const isMediumDevice = width >= 350 && width < 395
 export const isLargeDevice = width >= 395
+export const isVeryShortDevice = height < 500
```

Comment:
i think we need to use some common devices classifications 
for example, in CSS grid systems devices are large, medium, small and extra small
need to find how non-standard aspect ratios are categorized.

i do not require this change now, just create an issue for that and put in onto the Icebox please